### ISSUE TITLE
Error on import

### ISSUE THREAD

**Issue by sofroniewn** [0x00b1/napari-features#6]
Howdy

I was just trying to get this going but ran into a problem with translation strings on import

```python-traceback
In [1]: import napari_features
---------------------------------------------------------------------------
FileNotFoundError                         Traceback (most recent call last)
<ipython-input-1-19d276bb38fe> in <module>
----> 1 import napari_features

~/opt/anaconda3/lib/python3.7/site-packages/napari_features/__init__.py in <module>
      1 __version__ = "0.1.2"
      2
----> 3 from ._features import napari_experimental_provide_function

~/opt/anaconda3/lib/python3.7/site-packages/napari_features/_features.py in <module>
      6 from ._dock_widget import DockWidget
      7 from .generator import Generator
----> 8 from ._feature_selection_widget import FeatureSelectionWidget
      9
     10

~/opt/anaconda3/lib/python3.7/site-packages/napari_features/_feature_selection_widget.py in <module>
     11 resource_filename = pkg_resources.resource_filename(napari_features.__name__, "./data/translations/en.json")
     12
---> 13 with open(resource_filename, "r") as fp:
     14     FEATURES = json.load(fp)
     15

FileNotFoundError: [Errno 2] No such file or directory: '/Users/nsofroniew/opt/anaconda3/lib/python3.7/site-packages/napari_features/./data/translations/en.json'
```

I might be missing a translation pack here or something. I'll look into it, but would probably be good if it was either an explicit dependency or if we could skip it if not found

**Comment by sofroniewn** [comment]
Actually looks like it is something that didn't get included in the pip distribution and needs to be added to manifest https://github.com/0x00b1/napari-features/blob/024615b91ce296ddbb653cdae01f905a791d4e19/MANIFEST.in

**Comment by haesleinhuepf** [comment]
Hi @sofroniewn ,

I get the same error on my system, but the file is there. I presume, the issue is the "/./" in the path... On Windows the error looks a bit more weird because of too many backslashes, see #4

**Comment by sofroniewn** [comment]
Note i only get the error on pip install from PyPI - if I clone the repo and install from source it works fine

**Comment by haesleinhuepf** [comment]
```
pip install .
```
Freezes my terminal.

```
pip install -e .
```
Works fine.

**Comment by 0x00b1** [comment]
This should be fixed in 0.1.4 (available from PyPI)!
